File: librarymanagementsystem-frontend/src/app/features/models/responses/materials/created-material-response.ts
export interface ApiResponse<T>{
	message?: string;
	data: T;
}

export interface CreatedMaterialResponse {
	id: string;
	name: string;
	description: string;
	punishmentAmount: number;
	isBorrowable: boolean;
	borrowDay: number;
    createdDate: Date;
}

File: librarymanagementsystem-backend/src/librarymanagementsystembackend/Application/Features/Materials/Commands/Create/CreatedMaterialResponse.cs
using NArchitecture.Core.Application.Responses;

namespace Application.Features.Materials.Commands.Create;

public class CreatedMaterialResponse : IResponse
{
    public Guid Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public decimal? PunishmentAmount { get; set; }
    public bool IsBorrowable { get; set; }
    public byte BorrowDay { get; set; }
}